includes/Media.php

```diff
@@ -51,6 +51,13 @@ class Media {
 	 */
 	const STORY_SQUARE_IMAGE_SIZE = 'web-stories-poster-square';
 
+	/**
+	 * Name of size used in media library.
+	 *
+	 * @var string
+	 */
+	const STORY_THUMBNAIL = 'web_stories_thumbnail';
```

Comment:
Nitpick, but can we prefix this with `_IMAGE_SIZE` for consistency?

Reply:
Fixed https://github.com/google/web-stories-wp/pull/837/commits/304f8bccdc4bc5c9e643604307822dcb6a962b12

Reply:
Thanks!